implementations/rust/ockam/ockam_command/src/tcp/inlet/create.rs

```diff
@@ -203,7 +203,21 @@ impl CreateCommand {
 
         opts.terminal
             .stdout()
-            .plain(
+            .plain(if let Some(last_failure) = inlet.last_failure.as_deref() {
+                fmt_err!(
```

Comment:
Since this error it's not really a failure that forces the user to redo any action or do anything about it, I would use `fmt_warn`